roles/basic_prog_pkgs/tasks/main.yml

```diff
@@ -5,27 +5,10 @@
     name: '{{ basic_prog_pkgs_intro_development }}'
     state: latest
 
-- name: Copy idlerc prefs
-  ansible.builtin.copy:
-    src: .idlerc
-    dest: '{{ item.homedir }}'
-    owner: '{{ item.uid }}'
-    group: '{{ item.gid }}'
-    directory_mode: "0750"
-    mode: "0640"
-  loop: "{{ real_users }}"
-- name: Copy mimeapps Idle pref
-  ansible.builtin.copy:
-    src: mimeapps.list
-    dest: '{{ item.homedir }}/.config/'
-    owner: '{{ item.uid }}'
-    group: '{{ item.gid }}'
-    mode: "0640"
-    force: no
-  loop: "{{ real_users }}"
-- name: Set Idle pref in existing mimeapps
-  ansible.builtin.lineinfile:
-    path: '{{ item.homedir }}/.config/mimeapps.list'
-    regexp: '^text/x-python='
-    line: 'text/x-python=idle-python3.6.desktop;gedit.desktop;pluma.desktop'
+- name: Install intro pip packages
+  ansible.builtin.pip:
+    name: '{{ basic_prog_pkgs_pip }}'
+    state: present
```

Comment:
```suggestion
  ansible.builtin.pip:
    name: '{{ basic_prog_pkgs_pip }}'
    state: present
    extra_args: --user
```

Reply:
Do we set `PATH` stuff correctly to handle passing `--user`?

Reply:
Pip appears to be handling this correctly for a user install, perhaps since https://github.com/pypa/pip/pull/7002. Not sure I understand the PATH reference, but the user install is writing to .local/bin, which does not appear to be in PATH

Reply:
Hmm yeah, not having `~/.local/bin` in `$PATH` is probably fine for now with the tool(s) we've got at the moment. But it's probably something to be aware of in the future.